src/FeedsCleanup/packages.config

```diff
@@ -0,0 +1,6 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="Microsoft.Web.Xdt" version="2.1.0" targetFramework="net452" />
+  <package id="NDesk.Options" version="0.2.1" targetFramework="net452" />
```

Comment:
BTW eventually this utility should move to another repo anyway - DnxTools - but no hurry on that. That way it'll avoid re-building our tools every time this repo runs.